examples/jsm/controls/OrbitControls.js

```diff
@@ -124,6 +124,13 @@ class OrbitControls extends EventDispatcher {
 
 		};
 
+		this.removeKeyEvents = function () {
+
+			domElement.removeEventListener( 'keydown', onKeyDown );
+			this._domElementKeyEvents = null;
```

Comment:
I think this should be:
```js
this._domElementKeyEvents.removeEventListener( 'keydown', onKeyDown );
this._domElementKeyEvents = null;
```
You are referring to `domElement` which is not the DOM element for key events.

Reply:
You're right - thank you. I'll update this unless you get to it first.

Reply:
Done!

I just wonder if `removeKeyEvents()` is a good name (next to `listenToKeyEvents()`). 

@mrdoob What are your thoughts about this?

Reply:
Yeah, my first thought was "unlistenToKeyEvents()" so that's another option.

Reply:
I like this name even more. I was at first unsure about the "unlisten" term, but after read [this](https://ell.stackexchange.com/questions/214085/is-there-any-word-for-unlisten-negative-of-listen), I'm convinced it's good 😊 .